protocol/protos/NetRemoteDataStreamingService.proto

```diff
@@ -0,0 +1,12 @@
+
+syntax = "proto3";
+
+package Microsoft.Net.Remote.Service;
+
+import "NetRemote.proto";
+import "NetRemoteWifi.proto";
+
+service NetRemoteDataStreaming
+{
+    rpc WifiDataStreamUpload (stream Microsoft.Net.Remote.Wifi.WifiDataStreamUploadData) returns (Microsoft.Net.Remote.Wifi.WifiDataStreamUploadResult);
```

Comment:
Is there any reason this needs to be wifi specific? Calling this on a connection using a non-wifi adapter would still work fine, right? The same goes to all the proposed new types that reference `Wifi`. I think we can make these core streaming types independent of wifi.